screens/EpisodeInputScreen.tsx

```diff
@@ -0,0 +1,398 @@
+      recordingDay = msApart / DAY_IN_MS + 1;
+    }
+    if (todayEpisodes === null) {
+      await AsyncStorage.setItem(
+        STORAGE_KEYS.daysEpisodes(recordingDay),
+        JSON.stringify([])
+      );
+    } else {
+      setEpisodes(JSON.parse(todayEpisodes));
+    }
+  };
+
+  useEffect(() => {
+    onMount();
+  }, []);
+
+  return (
+    <View style={containerStyles.container}>
+      <View style={containerStyles.titleContainer}>
+        <Text style={styles.title}>Create an episode</Text>
+        <Icon
+          name="three-bars"
+          size={30}
+          color="#000"
+          onPress={() => {
+            navigation.navigate("EpisodeDisplay", { episodes });
+          }}
+        />
+      </View>
+      <View style={containerStyles.inputContainer}>
+        <View style={{ flex: 0.5 }} />
+        <View style={styles.input}>
+          <Text style={styles.inputHeader}>Episode Name</Text>
+          <TextInput
+            style={{ ...styles.inputContent, ...styles.textInput }}
+            maxLength={70}
+            onChangeText={setEpisodeName}
+            value={episodeName}
+            editable
+          />
+        </View>
+        <View style={styles.input}>
+          <Text style={styles.inputHeader}>{"Duration"}</Text>
+          <View style={{ ...styles.inputContent, ...styles.timePickers }}>
+            <TimePicker
+              style={styles.timePickerContainer}
+              time={startTime}
+              isVisible={isStartDatePickerVisible}
+              label="Start time"
+              setTime={setStartTime}
+              setVisibility={setStartDatePickerVisible}
+            />
+            <View style={{ flex: 10 }} />
+            <TimePicker
+              style={styles.timePickerContainer}
+              time={endTime}
+              isVisible={isEndDatePickerVisible}
+              label="End time"
+              setTime={setEndTime}
+              setVisibility={setEndDatePickerVisible}
+            />
+          </View>
+        </View>
+        <View style={styles.input}>
+          {/* <View style={{ flex: 1, flexDirection: "row" }}> */}
+          <Text style={styles.inputText}>
+            Persons involved{" "}
+            <Text style={{ ...styles.inputText, ...styles.greyText }}>
+              (Optional)
+            </Text>
+          </Text>
+
+          {/* </View> */}
+          <TextInput
+            style={{ ...styles.inputContent, ...styles.textInput }}
+            maxLength={70}
+            onChangeText={setInitials}
+            value={initials}
+            editable
+          />
+        </View>
+        <View style={{ flex: 0.5 }} />
+      </View>
+      <View style={containerStyles.buttonsContainer}>
+        <Button
+          disabled={!!hasError}
+          onPress={hasError ? () => {} : createEpisode}
+          style={{
+            ...(hasError ? styles.buttonGrey : styles.buttonBlue),
+            ...styles.button,
+          }}
+        >
```

Comment:
I know we have a lot of buttons that are like this, might make sense to make a component that is like "disablablebutton", but in a future pr